packages/components/src/components/dialog/dialog.stories.ts

```diff
@@ -154,16 +168,24 @@ export const ExtendedFooter = {
   },
   render: () => {
     return html`
+      <style>
+        @media (max-width: 414px) {
+          sd-dialog::part(footer) {
+            flex-direction: column;
+          }
+        }
+      </style>
+
       <div style="height: 40vh;">
         <sd-dialog open="" id="extended-footer"
           ><p class="sd-paragraph">
             Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
             phasellus dui vel id. Velit in sed.
           </p>
           <h4 slot="headline" class="sd-headline sd-headline--size-3xl">Lorem Ipsum</h4>
+          <sd-button variant="secondary" slot="footer" class="w-full">Label</sd-button>
           <sd-button slot="footer" class="w-full">Label</sd-button>
-          <sd-button variant="secondary" slot="footer" class="w-full">Label</sd-button></sd-dialog
-        >
+        </sd-dialog>
```

Comment:
@Vahid1919 I know that you did not change anything in the `PreventClosing` story but on this version published by chromatic, the buttons are not working anymore. can you pls have a look into it and either fix it or open up a bug ticket?!

Reply:
It seems like the buttons don't work when you are coming from the main story. Reloading it seems to make it work. I will write a ticket for this and see what we can do.